app/src/main/java/org/p2p/wallet/striga/presetpicker/mapper/StrigaItemCellMapper.kt

```diff
@@ -18,16 +18,13 @@ import org.p2p.wallet.striga.signup.model.StrigaSourceOfFunds
 class StrigaItemCellMapper {
 
     fun buildCellModels(items: List<StrigaPresetDataItem>, selectedItem: StrigaPresetDataItem?): List<AnyCellItem> {
-        val itemsWithoutSelected = items.filter { it != selectedItem }
         return buildList {
-            if (selectedItem != null) {
+            if (selectedItem?.getName()?.isNotEmpty() == true) {
```

Comment:
what does this mean? why we check name for emptyness?

Reply:
I just write that, if name is empty, than selected item is null x D

Reply:
I can make getName nullable

Reply:
so just check selectedItem != null?

Reply:
it's not null anyway. but item.country could be null )